cypress/integration/C_0001Sessions_paid_spec.js

```diff
@@ -0,0 +1,7 @@
+describe("Sessions Paid", () => {
+    it("Pays for session", () => {
+        cy.login({ email: "c@lvin.com", password: "password" })
```

Comment:
Extract these to separate variables for easier reuse and changing.

Reply:
sure